src/tests/system/tests/test_pysss_nss_idmap.py

```diff
@@ -0,0 +1,345 @@
+"""
+SSSD Python Client identity Lookups
+
+:requirement: Python binding of libsss_nss_idmap.so
+"""
+
+from __future__ import annotations
+
+import ast
+
+import pytest
+from sssd_test_framework.roles.ad import AD
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.roles.samba import Samba
+from sssd_test_framework.topology import KnownTopology, KnownTopologyGroup
+
+
+def get_sid_name(provider: GenericProvider):
```

Comment:
Generally, we haven't been adding functions to the test file. The placement of this method makes sense, the other option is to the framework. This is an FYI; it makes the most sense and improves the readability of the test code.

Reply:
Hi,

yes, I agree, those two are quite specific to the test, especially `run_pysss_nss_idmap`.

bye,
Sumit